### DISCUSSION TITLE
How to get swiper works on Angular 11 with ssr????? (swiper 6.8.4)

### DISCUSSION THREAD

**Discussion by erqk** [nolimits4web/swiper#5088] [Q&A] [answered]
Can I know how to get swiper working when I'm using Angular Universal? I can't get it to work when I followed the demo code.

Angular version: 11
Swiper version: 6.8.4

**Comment by erqk** [comment, marked as answer]
Sorry...I've my angular ssr setting file goes wrong. I fixed it and all works fine!

**Reply by zer0-um** [reply]
Hi @erqk can you provide an example how did you make it work with angular 11? I can't seem to do so

**Reply by erqk** [reply]
Hi, in my case I have trouble when installing Angular Universal, and when I correct it and reinstall, realize that the Angular code generation didn't know there is generated code exists. So my src/main.ts has duplicate code inside. I remove them then all works again.

I guess your problem is not my case:

If you're using angular 11, make sure you install swiper version 6. Since the latest version (which is 7 currently) is support only angular 12+, you can check out more on their website - Migration guide to swiper 7
